js/main.js

```diff
@@ -24,11 +24,12 @@ fetch('./js/fonts.json')
   .then((r) => r.json())
   .then((fontsObject) => {
     // create visual list for fonts
-    for (font of fontsObject.items) {
+    for ([index, font] of fontsObject.items.entries()) {
       let fontFamily = font.family;
       let li = document.createElement('li');
       li.classList.add('font');
       li.setAttribute('data-value', fontFamily);
+      li.setAttribute('tabindex', index+100);
```

Comment:
it is better NOT to be explicit/specific about tabindex here, all of them should have `tabindex=0`. 

> tabindex="0" means that the element should be focusable in sequential keyboard navigation, but its order is defined by the document's source order.

[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

imagine if the list item contains other "tabable" elements like an anchor ( a like or favorite button), if you specify an incremental `tabindex` tabs will jumb from `li` to another skipping any other "tabable" elements inside it.

to prove the point, add these three lines of code and give it a try. 



```js
      li.style.fontFamily = fontFamily;
      let a = document.createElement('a');
      a.innerText = ' like';
      a.href = '#';
      li.appendChild(a);
``` 

and as a side effect of using 0, you no longer need the destructure  [here](https://github.com/elharony/Google-Fonts-Previewer/pull/2/files#diff-77ab163850ef0fa98d6a9ee9fb6bbcafR27)

Reply:
Done, Thank You!